account_analytic_distribution_manual/hooks.py

```diff
@@ -157,6 +157,18 @@ def post_init_hook(cr, registry):
             new_distribution = DistributionManual.create(distribution_manual_val)
             distribution_map[tag_id] = new_distribution
             all_tag_ids.append(tag_id)
+            # Define the value of manual_distribution_id in the line items
+            update_sql = f"""
+            UPDATE account_analytic_line
+            SET manual_distribution_id = {new_distribution.id}
+            WHERE manual_distribution_id IS NOT NULL
+            ANBD id IN (
```

Comment:
This contains a typo and is not correct, as `manual_distribution_id` will always be NULL in this state.

Reply:
Why not using the same expression as the migration script?